packages/react-ui/internal/InputLikeText/InputLikeText.tsx

```diff
@@ -81,6 +82,15 @@ export class InputLikeText extends React.Component<InputLikeTextProps, InputLike
     if (this.dragging || !node) {
       return;
     }
+    if (
+      isIE11 &&
+      Array.from(document.body.querySelectorAll('[data-rendered-container-id]')).some(
+        containsTargetOrRenderContainer(node),
+      )
```

Comment:
Здесь тогда больше подойдет другая функция из `listenFocusOutside`, чтобы на детали реализации рендер-контейнеров не завязываться.

```suggestion
      findRenderContainer(node, document.body)
```

Reply:
Ну, мы всё равно завязываемся на эти детали, просто опосредованно)
Но я согласен, что лучше использовать уже готовую функцию.